packages/mgt-chat/src/components/UnsupportedContent/UnsupportedContent.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import React from 'react';
+import { registerAppIcons } from '../styles/registerIcons';
+import { ArrowSquareUpRight24Regular } from '@fluentui/react-icons';
+registerAppIcons();
```

Comment:
registerAppIcons is "above" this in the Chat component IIRC, is this needed here because of the fact that this is rendered vai a renderToString call?

Reply:
I've re-tested, it's not needed. Removing it.